src/docker-entrypoint.sh

```diff
@@ -6,93 +6,94 @@ set -e
-if [ ! -z $STATIC_EXTENSIONS ]; then
+if [ -n "$STATIC_EXTENSIONS" ]; then
   sed -z -E -i 's|#SecAction.{6}id:900260.*\x27tx\.static_extensions=/[[:lower:][:space:]/.]*\x27\"|SecAction \\\n  \"id:900260, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:\x27tx.static_extensions='"$STATIC_EXTENSIONS"'\x27\"|' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
 # Block request if number of arguments is too high
-if [ ! -z $MAX_NUM_ARGS ]; then
+if [ -n "$MAX_NUM_ARGS" ]; then
   sed -z -E -i 's/#SecAction.{6}id:900300.*tx\.max_num_args=255\"/SecAction \\\n  \"id:900300, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:tx.max_num_args='"$MAX_NUM_ARGS"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
 # Block request if the length of any argument name is too high
-if [ ! -z $ARG_NAME_LENGTH ]; then
+if [ -n "$ARG_NAME_LENGTH" ]; then
   sed -z -E -i 's/#SecAction.{6}id:900310.*tx\.arg_name_length=100\"/SecAction \\\n \"id:900310, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:tx.arg_name_length='"$ARG_NAME_LENGTH"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
 # Block request if the length of any argument value is too high
-if [ ! -z $ARG_LENGTH ]; then
+if [ -n "$ARG_LENGTH" ]; then
   sed -z -E -i 's/#SecAction.{6}id:900320.*tx\.arg_length=400\"/SecAction \\\n  \"id:900320, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:tx.arg_length='"$ARG_LENGTH"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
 # Block request if the total length of all combined arguments is too high
-if [ ! -z $TOTAL_ARG_LENGTH ]; then
+if [ -n "$TOTAL_ARG_LENGTH" ]; then
   sed -z -E -i 's/#SecAction.{6}id:900330.*tx\.total_arg_length=64000\"/SecAction \\\n  \"id:900330, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n  t:none, \\\n   setvar:tx.total_arg_length='"$TOTAL_ARG_LENGTH"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
 # Block request if the total length of all combined arguments is too high
-if [ ! -z $MAX_FILE_SIZE ]; then
+if [ -n "$MAX_FILE_SIZE" ]; then
   sed -z -E -i 's/#SecAction.{6}id:900340.*tx\.max_file_size=1048576\"/SecAction \\\n  \"id:900340, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:tx.max_file_size='"$MAX_FILE_SIZE"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
 # Block request if the total size of all combined uploaded files is too high
-if [ ! -z $COMBINED_FILE_SIZES ]; then
+if [ -n "$COMBINED_FILE_SIZES" ]; then
   sed -z -E -i 's/#SecAction.{6}id:900350.*tx\.combined_file_sizes=1048576\"/SecAction \\\n  \"id:900350, \\\n   phase:1, \\\n   nolog, \\\n   pass, \\\n   t:none, \\\n   setvar:tx.combined_file_sizes='"$COMBINED_FILE_SIZES"'\"/' /etc/modsecurity.d/owasp-crs/crs-setup.conf
 fi 
 
-if [ $WEBSERVER = "Apache" ]; then
-  if [ ! -z $PROXY ]; then
-    if [ $PROXY -eq 1 ]; then
+if [ "$WEBSERVER" = "Apache" ]; then
+  if [ -n "$PROXY" ]; then
+    if [ "$PROXY" -eq 1 ]; then
```

Comment:
We could replace both lines by simply
```bash
  if [ "$PROXY" = "1" ]; then
```
Would you care adapting also that?

Reply:
Sure, done!